.github/workflows/build.yml

```
@@ -95,6 +96,7 @@ jobs:
           mkdir -p northstar/R2Northstar/plugins
           mkdir -p northstar/bin/x64_retail
           mv -v northstar-launcher/DiscordRPC.dll northstar/R2Northstar/plugins
+          mv -v northstar-launcher/discord_game_sdk.dll northstar/R2Northstar/plugins
```

This dll needs to be in the main directory, as thats where discord wants to load it from.
```suggestion
          mv -v northstar-launcher/discord_game_sdk.dll northstar
```

---

Yeah that fixed it.

---

Oops, I had thought we loaded plugins with LOAD_WITH_ALTERED_SEARCH_PATH, but [apparently not](https://github.com/R2Northstar/NorthstarLauncher/commit/13dcc7d9307b160131ed80832efda99001aa6c6d#diff-0dd86527f62e938ec829226a56cfd38a397b0b23676ab02531cb670bdfb4afcaR164).